css/index.css

```diff
@@ -106,3 +107,148 @@ ul li a svg path {
 ul li a svg path:hover {
```

Comment:
This doesn't look good at all, try to give the anchor tag a class and style the class instead of writing like 4 elements